app-backend/common/src/main/scala/AWSBatch.scala

```diff
@@ -0,0 +1,45 @@
+package com.azavea.rf.common
+
+import java.util.UUID
+
+import com.typesafe.scalalogging.LazyLogging
+
+import com.amazonaws.services.batch.AWSBatchClientBuilder
+import com.amazonaws.services.batch.model.SubmitJobRequest
+
+import scala.collection.immutable.Map
+import scala.collection.JavaConversions._
+
+
+/** Submits jobs to AWS Batch for processing */
+trait AWSBatch extends RollbarNotifier with LazyLogging {
+
+  val awsbatchConfig = Config.awsbatch
+
+  val batchClient = AWSBatchClientBuilder.defaultClient()
+
+  def submitJobRequest(jobDefinition: String, jobQueueName: String, parameters: Map[String, String]) = {
+    val jobRequest = new SubmitJobRequest()
+      .withJobName(jobDefinition)
```

Comment:
It might be good to add a relevant ID to this in some consistent way (so that we can easily `grep` the output of `aws batch list-jobs`). I did something not particularly well-thought-out with the scheduled Landsat and Sentinel jobs: `jobStagingImportLandsat8Scenes-2017-09-21`, where `2017-09-12` is the date of imagery we're querying for.